mod-orders-storage/examples/order_invoice_relationship_collection.sample

```diff
@@ -0,0 +1,21 @@
+{
+  "orderInvoiceRelationships":
+    [
+     {
+        "id": "2A1030d4-C4EC-5A66-8EfC-3fc8e608e2De",
+        "purchaseOrderId": "0610be6d-0ddd-494b-b867-19f63d8b5d6d",
+        "invoiceId": "c0d08448-347b-418a-8c2f-5fb50248d67e",
```

Comment:
Please remove comma after last member of each item

Reply:
Done